components/gripper/gripper.go

```diff
@@ -89,3 +90,29 @@ func FromDependencies(deps resource.Dependencies, name string) (Gripper, error)
 func NamesFromRobot(r robot.Robot) []string {
 	return robot.NamesByAPI(r, API)
 }
+
+// MakeModel is a helper function that creates a zero DoF Model for a gripper from a list of its geometries.
+func MakeModel(name string, geometries []spatialmath.Geometry) (referenceframe.Model, error) {
```

Comment:
@randhid it ended up being much more code duplication not making this public.  There isn't a good way to make a model with a single geometry right now.  I think its likely that this will be a semi-long term solution too so making it visible to module implementers is possibly useful for local testing, etc.

Reply:
That's fine.